lib/glific/wa_managed_phones.ex

```diff
@@ -52,6 +53,28 @@ defmodule Glific.WAManagedPhones do
   @spec get_wa_managed_phone!(non_neg_integer()) :: WAManagedPhone.t()
   def get_wa_managed_phone!(id), do: Repo.get!(WAManagedPhone, id)
 
+  @doc """
+  Gets a single wa_managed_phone.
+
+  Raises `Ecto.NoResultsError` if the Wa managed phone does not exist.
+
+  ## Examples
+
+      iex> get_wa_managed_phone(123)
+      %WAManagedPhone{}
+
+      iex> get_wa_managed_phone(456)
+      ** (Ecto.NoResultsError)
```

Comment:
This actually give nil and doesn't raise exception. Can we change this?.
Also same thing at the start of doc